src/services/auth.services.ts

```diff
@@ -8,21 +8,30 @@ export default class AuthServices {
   /** The Auth API URL. */
   static authApiUrl = sessionStorage.getItem('AUTH_API_URL')
 
-  /** Creates an NR Affiliation */
-  static async createNRAffiliation (orgIdentifier: number, affiliation: CreateNRAffiliationRequestBody): Promise<any> {
-    const url = `${this.authApiUrl}/orgs/${orgIdentifier}/affiliations?newBusiness=true`
-    return axios.post(url, affiliation)
+  /** Creates an NR affiliation. */
+  static async createNrAffiliation (accountId: number, nr: any): Promise<any> {
+    const url = `${this.authApiUrl}/orgs/${accountId}/affiliations?newBusiness=true`
+    const requestBody: CreateNRAffiliationRequestBody = {
+      businessIdentifier: nr.nrNum,
+      phone: nr.applicants?.phoneNumber || '',
+      email: nr.applicants?.emailAddress || ''
+    }
+
+    return axios.post(url, requestBody)
+      .catch(error => error?.response)
```

Comment:
Now this method never returns an exception, and the caller can check the response status.